src/game.js

```diff
@@ -128,7 +128,7 @@ var Game = Class.create({
 			},
 			ui: {
 				dash: {
-					materialize_overload: "Overload! Maximum number of units controlled",
+					materialize_overload: "Overload! Maximum number of units controlled reached.",
```

Comment:
This is short on purpose, as it won't fit the UI otherwise.

Reply:
Ok :)